### PR TITLE
BUGFIX: SD-384 apply 'follow pointer' to combo chart

### PR DESCRIPTION
**Pull request by naibinh**
[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`

# Related PRs
- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2183
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/1879

# Related Jira tasks
- SD-384: https://jira.intgdc.com/browse/SD-384

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3217/ : SUCCESS in 13m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3973/ : SUCCESS in 14s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1719/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1063/ : SUCCESS in 31m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3974/ : SUCCESS in 14s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-913.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1067/ : SUCCESS in 33m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3984/ : SUCCESS in 10s

File: src/components/visualizations/chart/highcharts/test/customConfiguration.spec.ts [hunk 1]
```diff
@@ -542,33 +542,39 @@ describe("getCustomizedConfiguration", () => {
...
-            const result = getCustomizedConfiguration({
-                ...chartOptions,
-                actions: { tooltip: true },
-                data: getData([{ y: 100 }, { y: 101 }]),
-                type: VisualizationTypes.COLUMN,
-                yAxisProps: {
-                    max: 50,
-                },
-            });
+        it.each([[VisualizationTypes.COLUMN], [VisualizationTypes.BAR], [VisualizationTypes.COMBO]])(
```
Comment (ptquang86) [inline src/components/visualizations/chart/highcharts/test/customConfiguration.spec.ts]: should we reuse supportedTooltipFollowPointerChartTypes here?

Comment (naibinh) [reply src/components/visualizations/chart/highcharts/test/customConfiguration.spec.ts]: Fixed.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3235/ : FAILURE in 9m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4008/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3237/ : SUCCESS in 5m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4012/ : SUCCESS in 15s

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1735/ : SUCCESS in 19m 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3244/ : SUCCESS in 3m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/886/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/910/ : FAILURE in 27m 55s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1737/ : SUCCESS in 19m 57s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3247/ : SUCCESS in 3m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/887/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/911/ : FAILURE in 27m 26s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1739/ : SUCCESS in 14m 07s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3250/ : SUCCESS in 5m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/888/ : SUCCESS in 46s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/912/ : FAILURE in 24m 14s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1740/ : SUCCESS in 23m 34s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3252/ : SUCCESS in 12m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/889/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/913/ : FAILURE in 34m 11s

**Comment by naibinh** [comment]
extended test - examples

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3256/ : SUCCESS in 8m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4037/ : SUCCESS in 27s

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-913.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-913.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1094/ : SUCCESS in 35m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4036/ : SUCCESS in 2m 32s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1748/ : SUCCESS in 23m 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3262/ : SUCCESS in 3m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/893/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/922/ : SUCCESS in 26m 41s

**Pull request merged**